src/app/components/navbar/navbar.component.ts

```diff
@@ -94,7 +92,6 @@ export class NavbarComponent implements OnInit, OnDestroy {
 
     private async onTemplateChanges(template: TemplateApiObject | null) {
         if (template == null) {
-            this.extraTabs = [];
```

Comment:
This should be replaced with setting both extra tabs to empty lists (or trigger both update functions)